src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java

```diff
@@ -6,26 +6,45 @@
 public class AdsFandomContent {
 
   public static final Map<String, String> SLOT_SELECTORS;
+  public static final Map<String, String> GPT_SLOT_SELECTORS;
 
   public static final String TOP_LEADERBOARD = "TOP_LEADERBOARD";
   public static final String BOTTOM_LEADERBOARD = "BOTTOM_LEADERBOARD";
+  public static final String BOTTOM_LEADERBOARD_DESKTOP = "BOTTOM_LEADERBOARD_DESKTOP";
+  public static final String BOTTOM_LEADERBOARD_MOBILE = "BOTTOM_LEADERBOARD_MOBILE";
   public static final String TOP_BOXAD = "TOP_BOXAD";
+  public static final String TOP_BOXAD_DESKTOP = "TOP_BOXAD_DESKTOP";
+  public static final String TOP_BOXAD_MOBILE = "TOP_BOXAD_MOBILE";
   public static final String INCONTENT_BOXAD = "INCONTENT_BOXAD";
+  public static final String INCONTENT_BOXAD_MOBILE = "INCONTENT_BOXAD_MOBILE";
   public static final String BOTTOM_BOXAD = "BOTTOM_BOXAD";
-  public static final String GPT_TOP_LEADERBOARD = "gpt-top-leaderboard";
-  public static final String GPT_BOTTOM_LEADERBOARD = "gpt-bottom-leaderboard-desktop";
-  public static final String GPT_BOTTOM_LEADERBOARD_MOBILE = "gpt-bottom-leaderboard-mobile";
 
   static {
     SLOT_SELECTORS = new HashMap<>();
     SLOT_SELECTORS.put(TOP_LEADERBOARD, "div[id$='TOP_LEADERBOARD_0__container__']");
     SLOT_SELECTORS.put(BOTTOM_LEADERBOARD, "div[id$='BOTTOM_LEADERBOARD_0__container__']");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)